controller/viewcontroller.php

```diff
@@ -191,13 +199,20 @@ private function getTemplateParams() {
 		$defaultColor = $this->config->getAppValue('theming', 'color', '#0082C9');
 		$canSharePublicLink = $this->config->getAppValue('core', 'shareapi_allow_links', 'yes');
 
+		$isSpreedAvailable = $this->appManager->isEnabledForUser('spreed', null);
```

Comment:
should be done with events and be handled in the other app then

Reply:
Same, currently not possible.

Reply:
```php
$this->dispatcher->dispatch('<event name>', new GenericEvent($data));
```

anyway…

Reply:
> currently not possible

What I meant by that is that the Calendar app can't be extended by the Spreed app. Triggering an event wouldn't help at all since there's nothing to be done by Spreed.

Reply:
Spreed could extend the array before it's send to the Template, anyway

Reply:
That's indeed true for loading the template variables. But right now there is no way for the spreed app to register a new tab in the event editor.